lib/src/components/Badge/Badge.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { ForwardRefRenderFunction } from "react";
+import {
+  badges,
+  badgeWrapper,
+  badgeThemeClass,
+  badgeThemeVars
+} from "./badge.css";
+import { assignInlineVars } from "@vanilla-extract/dynamic";
+import { IBadgesProps } from "./badge.types";
+
+const BadgeComponent: ForwardRefRenderFunction<
+  HTMLSpanElement,
+  IBadgesProps
+> = (
+  {
+    color = "red",
+    textColor = "#fff",
+    hide = false,
+    shape = "rounded-circle",
+    position = "default",
+    label,
+    children,
+    ...props
+  },
+  ref
+) => {
+  const badgeStyle = badges({
+    shape,
+    visible: hide ? "hide" : "show",
+    position
+  });
+
+  const badgeWrapperStyle = badgeWrapper({});
+
+  return (
+    <span className={`${badgeWrapperStyle}`}>
+      {children}
+      <small
+        style={assignInlineVars({
+          [badgeThemeVars.badgeStyleColor]: color,
+          [badgeThemeVars.badgeStyleTextColor]: textColor
+        })}
+        className={`${badgeStyle}  ${badgeThemeClass}`}
+      >
```

Comment:
can we give this a role="status" by default?

reference: https://stackoverflow.com/a/68170644/9390649